docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,14 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+* Learn how to integrate with Eclipse:
+    * [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+    * [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+    * [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+* Learn how to intgrate with Intellij:
+    * [[Import an existing Maven project into Intellij|intellijMaven]]
+    * [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+* [[Learn about Lagom core concepts|CoreConcepts]]
```

Comment:
The way this is formatting looks a bit strange to me.

<img width="698" alt="screen shot 2017-02-22 at 2 11 08 pm" src="https://cloud.githubusercontent.com/assets/44385/23195640/cbe3ece2-f908-11e6-9b17-c4ff8bccc8e7.png">

I guess it's a bug in the stylesheet, which we could fix, but I also wonder if it would look better if the "Learn how..." headings were formatted as level 2 headings instead of bullets, and "Learn about Lagom core concepts" is a bit redundant with the "Next" link below it, so maybe we don't need it (or could replace it with something like 'or click "Next" to learn more about the core concepts in Lagom')